### PR TITLE
Submitting digiCamControl version 2.1.6.0

### PR DESCRIPTION
**Pull request by daymickcorr**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
yes

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
yes 
- [x] This PR only modifies one (1) manifest
yes 
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
yes
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
yes
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?
yes 
Note: `<path>` is the name of the directory containing the manifest you're submitting.
yes
-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103689)

### CODE REVIEW

**Comment by daymickcorr** [comment]
@microsoft-github-policy-service agree

File: manifests/d/dukus/digiCamControl/2.1.6.0/dukus.digiCamControl.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: dukus.digiCamControl
+PackageVersion: 2.1.6.0
+PackageLocale: en-US
+Publisher: dukus
+PackageName: digiCamControl
```
Comment (stephengillie) [inline manifests/d/dukus/digiCamControl/2.1.6.0/dukus.digiCamControl.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://digicamcontrol.com/
PublisherSupportUrl: https://sourceforge.net/projects/digicamcontrol/support
PackageName: digiCamControl
PackageUrl: https://digicamcontrol.com/download
```

File: manifests/d/dukus/digiCamControl/2.1.6.0/dukus.digiCamControl.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+PackageVersion: 2.1.6.0
+PackageLocale: en-US
+Publisher: dukus
+PublisherUrl: https://digicamcontrol.com/
+PublisherSupportUrl: https://sourceforge.net/projects/digicamcontrol/support
+PackageName: digiCamControl
+PackageUrl: https://digicamcontrol.com/download
+License: MIT License
+Copyright: Duka Istvan
+ShortDescription: digiCamControl
```
Comment (Trenly) [inline manifests/d/dukus/digiCamControl/2.1.6.0/dukus.digiCamControl.locale.en-US.yaml:14]: Please provide a better description of what digiCamControl is used for / does

File: manifests/d/dukus/digiCamControl/2.1.6.0/dukus.digiCamControl.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+PackageIdentifier: dukus.digiCamControl
+PackageVersion: 2.1.6.0
+PackageLocale: en-US
+Publisher: dukus
+PublisherUrl: https://digicamcontrol.com/
+PublisherSupportUrl: https://sourceforge.net/projects/digicamcontrol/support
+PackageName: digiCamControl
+PackageUrl: https://digicamcontrol.com/download
+License: MIT License
```
Comment (Trenly) [inline manifests/d/dukus/digiCamControl/2.1.6.0/dukus.digiCamControl.locale.en-US.yaml:12]:
```suggestion
License: MIT
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**